logger/logger.go

```diff
@@ -255,7 +256,16 @@ func (l *Logger) logWithLevel(logLvl LogLevel, data map[string]interface{}) {
 		data["_kvmeta"] = globalRouter.Route(data)
 	}
 
-	l.fLogger.formatAndLog(data)
+	// call any embedded outputs
+	routedLocally, err := embeddedoutputs.Route(data)
+	if err != nil {
+		l.ErrorD("embedded-output-error", map[string]interface{}{"error": err.Error()})
```

Comment:
not sure we should emit an error log. the other `Route` methods above don't report errors